verifier/credentialsConfig.go

```diff
@@ -68,24 +62,47 @@ func InitServiceBackedCredentialsConfig(repoConfig *config.ConfigRepo) (credenti
 func (cc ServiceBackedCredentialsConfig) fillStaticValues() {
 	for _, configuredService := range cc.initialConfig.Services {
 		logging.Log().Debugf("Add to service cache: %s", configuredService.Id)
-		cc.serviceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
+		err := common.GlobalCache.ServiceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
+		if err != nil {
+			logging.Log().Errorf("failed caching configured service in fillStaticValues(): %v", err)
+		}
 	}
 }
 
-func (cc ServiceBackedCredentialsConfig) fillCache(ctx context.Context) {
-	client := *(cc.configClient)
-	services, err := client.GetServices()
+func (cc ServiceBackedCredentialsConfig) fillCache(context.Context) {
```

Comment:
can we change the interface of this method if we done use the context?

Reply:
The context is needed in the function definition, because it's needed by chrono task scheduler. Did you mean to remove the context parameter? Or something else?

Reply:
Yes thats what I meant. But if its required then no problem